book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+## What is cultural change?
+
+Culture is the social environment in which you partake. 
```

Comment:
I like this definition, but I might expand it even further – maybe even address how "fuzzy" it is for people to pin down (and deeply controversional for the fields that study it). Can we customise this, or attune it specifically to the "culture" of academia?

I've seen culture defined as everything from "socially transmitted behavior patterns, arts, beliefs, institutions, and all other products of human work and thought typical of a population or community at a given time" (Webster) to a "way of life, especially in terms of customs or behaviors of a particular community or group within society" (Cambridge). The way that culture is defined within institutions is a big question in and of itself!

What about something like: 
```suggestion
Culture is defined in many ways, one of which is about the social environment in which you partake, or the customs and behaviours that define a group of people.
```

Reply:
Great points! I love the edit! 😄